codebuild_specs/aggregate_e2e_reports.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+env:
+  shell: bash
+phases:
+  build:
+    commands:
+      - cd ./scripts
+      - npm install -g ts-node
+      - npm install aws-sdk
```

Comment:
why isn't this not also a global install? why not combine line 8 and 9 so we don't have to run npm twice/

Reply:
@goldbez has more details on this; but I recall that ts-node needed to be installed globally to utilize it like in line 10, otherwise it was not being detected. It's likely both ts-node/aws-sdk were being installed in the same line until running into that issue